$NetBSD: patch-aj,v 1.2 2011/12/20 16:18:16 dholland Exp $

 - fix conflict with posix getline
 - include stdio.h preemptively to make sure the getline #define doesn't
   break it
 - const correctness demanded by recent gcc

--- getline.h.orig	1997-02-21 11:50:14.000000000 +0000
+++ getline.h
@@ -6,6 +6,9 @@
 #include "buttons.h"
 #include "bool.h"
 
+#include <stdio.h>
+#define getline getline_class
+
 class getline
   {public:
 
@@ -37,7 +40,7 @@ class getline
    bool    was_deactive;
    bool    is_single_char;
    
-   getline      (char name [],
+   getline      (const char name [],
                  win  *w,
                  char string [],
                  int  x,
